lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/utilities.scss

```diff
@@ -286,111 +233,154 @@
 
 // Top Margin
 
-.margin-top-xl {
-  margin-top: var(--space-xl);
-}
-
-.margin-top-lg {
-  margin-top: var(--space-lg);
-}
-
-.margin-top-md {
-  margin-top: var(--space-md);
-}
-
-.margin-top-sm {
-  margin-top: var(--space-sm);
-}
+.margin-top-xl { margin-top: var(--space-xl); }
+.margin-top-lg { margin-top: var(--space-lg); }
+.margin-top-md { margin-top: var(--space-md); }
+.margin-top-sm { margin-top: var(--space-sm); }
+.margin-top-xs { margin-top: var(--space-xs); }
+.margin-top-none { margin-top: 0; }
 
-.margin-top-xs {
-  margin-top: var(--space-xs);
-}
+// Bottom Margin
 
-.margin-top-none {
-  margin-top: 0;
-}
+.margin-bottom-xl { margin-bottom: var(--space-xl); }
+.margin-bottom-lg { margin-bottom: var(--space-lg); }
+.margin-bottom-md { margin-bottom: var(--space-md); }
+.margin-bottom-sm { margin-bottom: var(--space-sm); }
+.margin-bottom-xs { margin-bottom: var(--space-xs); }
+.margin-bottom-none { margin-bottom: 0; }
 
-// Bottom Margin
+// Right Margin
 
-.margin-bottom-xl {
-  margin-bottom: var(--space-xl);
-}
+.margin-right-xl { margin-right: var(--space-xl); }
+.margin-right-lg { margin-right: var(--space-lg); }
+.margin-right-md { margin-right: var(--space-md); }
+.margin-right-sm { margin-right: var(--space-sm); }
+.margin-right-xs { margin-right: var(--space-xs); }
+.margin-right-none { margin-left: 0; }
 
-.margin-bottom-lg {
-  margin-bottom: var(--space-lg);
-}
+// Left Margin
 
-.margin-bottom-md {
-  margin-bottom: var(--space-md);
-}
+.margin-left-xl { margin-left: var(--space-xl); }
+.margin-left-lg { margin-left: var(--space-lg); }
+.margin-left-md { margin-left: var(--space-md); }
+.margin-left-sm { margin-left: var(--space-sm); }
+.margin-left-xs { margin-left: var(--space-xs); }
+.margin-left-none { margin-left: 0; }
 
-.margin-bottom-sm {
-  margin-bottom: var(--space-sm);
-}
+// Elevations
+.elevation-1, .elevation-1.card { box-shadow: var(--elevation-1); }
```

Comment:
@Jeremy-Walton Any way we could just get this done with only the `.elevation-[#]` classes and drop `.elevation-[#].card`?